### ISSUE TITLE
File access woes

### ISSUE THREAD

**Issue by gingerbeardman** [saenzramiro/rambox#1154]
### Steps to reproduce

1. Use an SSD with trim
2. Use rambox
3. Wait

### Expected behavior

Should be normal operation

### Actual behavior

Excessive file access...

On my SSD this manifests as system waits as there are too many trims

If I leave rambox open all day I get 3 x 2 minute system pauses throughout the day.

system does not crash, but waits until drive has finishe processing.

if I do not leave rambox open I do not get any waits.

if I use franz there are far less waits. so I am back to franz 

### ENVIRONMENT

> Rambox: 0.5.12

> OS: 10.12.6

> Arch: x64

**Comment by benjezzy** [comment]
I've got the same issue on Windows 10

**Comment by gingerbeardman** [comment]
@benjezzy what SSD and what services do you have active in Rambox. I've kept my installation so we can try to figure out if it's a general Rambox issue or an issue with a particular service.

**Comment by benjezzy** [comment]
@gingerbeardman 
- I've got a Samsung 256GB NVMe PM951
- Services: Whatsapp, FB Messenger, Slack, Discord & Pushbullet.

**Comment by gingerbeardman** [comment]
Just because there's me comments doesn't mean it's not an issue

**Comment by benjezzy** [comment]
Agreed, I'm in the same situation as @gingerbeardman , stopped using Rambox because of this issue. Maybe that's how the app got it name?

**Comment by gingerbeardman** [comment]
stale but still an issue

**Comment by gingerbeardman** [comment]
stale but still an issue

**Comment by gingerbeardman** [comment]
stale but still an issue

**Comment by gingerbeardman** [comment]
stale but still an issue

**Comment by gingerbeardman** [comment]
stale but still an issue

**Comment by gingerbeardman** [comment]
stale but still an issue
